src/components/PeoplePage.tsx

```diff
@@ -1,29 +1,84 @@
+import { useEffect, useState } from 'react';
+import { useSearchParams } from 'react-router-dom';
+
 import { PeopleFilters } from './PeopleFilters';
 import { Loader } from './Loader';
 import { PeopleTable } from './PeopleTable';
+import { Person } from '../types';
+import { getPeople } from '../api';
+import { getFilteredPeople } from '../utils/GetFilteredPeople';
+import { getPeopleWithParents } from '../utils/GetPreparedPeople';
+import { getSortedPeople } from '../utils/GetSortedPeople';
 
 export const PeoplePage = () => {
+  const [people, setPeople] = useState<Person[]>([]);
+  const [loading, setLoading] = useState(false);
+  const [error, setError] = useState(false);
+  const [searchParams, setSearchParams] = useSearchParams();
+
+  const query = searchParams.get('query');
+  const centuries = searchParams.getAll('century');
+  const sex = searchParams.get('sex');
+  const sort = searchParams.get('sort');
+  const order = searchParams.get('order');
+
+  const filteredPeople = getFilteredPeople(people, {
+    centuries: centuries,
+    sex: sex,
+    query: query,
+  });
+
+  const sortedPeople = getSortedPeople(filteredPeople, sort, order);
+
+  useEffect(() => {
+    setLoading(true);
+    setError(false);
+
+    getPeople()
+      .then(humans => {
+        setPeople(getPeopleWithParents(humans));
+      })
+      .catch(() => setError(true))
+      .finally(() => setLoading(false));
+  }, []);
+
   return (
     <>
       <h1 className="title">People Page</h1>
 
       <div className="block">
         <div className="columns is-desktop is-flex-direction-row-reverse">
           <div className="column is-7-tablet is-narrow-desktop">
-            <PeopleFilters />
+            <PeopleFilters
```

Comment:
[ TASK ] : The sidebar with the filters should appear only when people are loaded.